ebean-core/src/main/java/io/ebeaninternal/server/persist/BatchPostExecute.java

```diff
@@ -45,4 +45,11 @@ public interface BatchPostExecute {
    * Add timing metrics for batch persist.
    */
   void addTimingBatch(long startNanos, int batch);
+
+  /**
+   * Undos generated properties.
+   */
+  default void onFailedUpdateUndoGeneratedProperties() {
```

Comment:
checkme: Rename to "onExecuteUndo(ex)"?

Are there possible uses cases in CallableSql/OrmUpdate/SqlUpdate?